### PR TITLE
Installer

### PR DESCRIPTION
**Pull request by lneir**
Jon, please review, this is work for installer.  Work is pretty much done...remaining work includes url config and final splash screen from design

### CODE REVIEW

File: minuet/symphony/Symphony/Symphony/config.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ﻿{
-  "url" : "https://dev6.symphony.com"
+  "url" : "https://beta.symphony.com"
```
Comment (jonmclachlan) [inline minuet/symphony/Symphony/Symphony/config.json:2]: Ok, so we'll be shipping pointing only to beta, for the moment.  More than reasonable.

File: minuet/symphony/Installer/Minuet-x86.aip [hunk 2]
```diff
@@ -0,0 +1,375 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<DOCUMENT Type="Advanced Installer" CreateVersion="11.7.1" version="11.7.1" Modules="enterprise" RootPath="." Language="en" Id="{B6EC4BDE-C578-4A57-A3D2-C9B0154B042C}">
+  <COMPONENT cid="caphyon.advinst.msicomp.ProjectOptionsComponent">
+    <ROW Name="HiddenItems" Value="SccmComponent;ActSyncAppComponent"/>
+  </COMPONENT>
+  <COMPONENT cid="caphyon.advinst.msicomp.MsiPropsComponent">
+    <ROW Property="AI_APP_ARGS" Value="/start-app=&quot;[#Symphony.pgx]&quot;"/>
+    <ROW Property="AI_BITMAP_DISPLAY_MODE" Value="0"/>
+    <ROW Property="AI_MINDOTNETVERSION" MultiBuildValue="DefaultBuild:3.5" ValueLocId="-"/>
```
Comment (jonmclachlan) [inline minuet/symphony/Installer/Minuet-x86.aip:9]: Nice. :)

File: minuet/symphony/Paragon/App.xaml.cs [hunk 3]
```diff
@@ -37,12 +37,12 @@ protected override void OnStartup(StartupEventArgs e)
...
             try
             {
-                Logger.Info("Paragon starting");
+                Logger.Info("Minuet starting");
 
                 // Create splash screen, if it is not explicitly disabled
                 if (!_suppressSplashScreen)
                 {
-                    var iconPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "window.ico");
+                    var iconPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "symphony-logo.png");
```
Comment (jonmclachlan) [inline minuet/symphony/Paragon/App.xaml.cs:45]: I assume this is the file at, minuet/symphony/Paragon/symphony-logo.png

Are we 100% sure that logo is the one we want? It look just slightly off from the usual.  Just being curious, maybe that subtle difference is intended :)

Comment (lneir) [inline minuet/symphony/Paragon/App.xaml.cs:45]: This is the icon in splashscreen - it is more or less placeholder until design provide new splash screen.

Comment (jonmclachlan) [inline minuet/symphony/Paragon/App.xaml.cs:45]: Sounds good.

**Pull request merged**